voorhoede/head-start
# fix: Cloudflare Pages function size

# Changes

- Adds postbuild logging for function size and total size for easier debugging (see below)
- Removes 404 function postbuild as it causes the `workers.api.error.script_too_large` (even though it's smaller than the permitted 1MiB); still we can live with a preview mode without a 404.

```
[post:function-sizes    ] 308K  functions/[locale]/[[path]].js
[post:function-sizes    ]  96K  functions/[locale]/[slug].js
...
[post:function-sizes    ] 136K  functions/robots.txt.js
[post:function-sizes    ] 2.9M  functions/
```

# Associated issue

Resolves #123 

# How to test

1. Open a preview page like https://fix-function-size.head-start.pages.dev/en/overview-page/
2. Verify preview mode works

# Checklist

- ~~I have added a decision log entry if the change affects the architecture or changes a significant technology~~
